tests/find/test-suite-1/test.default-index.js

```diff
@@ -357,4 +357,60 @@ testCases.push(function (dbType, context) {
       });
     });
   });
+  it('handles zero as a valid index value', function () {
```

Comment:
@nerumo It appears that the reason this is failing is that this test doesn't _return_ a promise; it merely calls `db.createIndex()` as a side effect. Thus is creates race conditions, and that's why we're seeing errors.

Reply:
I went ahead and made the fixup change; let's see if the tests pass...

Reply:
@nolanlawson ah true, I missed that. Tnx